lib/miq_expression/target.rb

```diff
@@ -19,7 +20,11 @@ def self.parse_params(field)
     rescue LoadError # issues for case sensitivity (e.g.: VM vs vm)
       parsed_params[:model_name] = nil
     end
-    parsed_params[:associations] = parsed_params[:associations].to_s.split(".")
+    parsed_params[:associations] = if parsed_params[:associations]
+                                     parsed_params[:associations][1..].split(".")
```

Comment:
ohhh I see you left the dot in there - I'm not sure - I feel like the parser should remove that (this is part of the reason I moved this into a method per subclass)

Reply:
that `\.?` adds a lot. Makes it very ambiguous between the left and right hand sides